compatibility/sandbox-migration/src/com/daml/Divulgence.scala

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml
+
+import akka.stream.Materializer
+import com.daml.platform.participant.util.ValueConversions._
+import com.daml.JsonProtocol._
+import com.daml.grpc.adapter.ExecutionSequencerFactory
+import com.daml.ledger.api.v1.event.CreatedEvent
+import com.daml.ledger.api.v1.value.Identifier
+import com.daml.ledger.client.LedgerClient
+import com.daml.ledger.client.configuration.{
+  CommandClientConfiguration,
+  LedgerClientConfiguration,
+  LedgerIdRequirement
+}
+import spray.json.RootJsonFormat
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object Divulgence {
+
+  private final case class Result(
+      oldPrivateAssets: Seq[Application.ContractResult],
```

Comment:
The terminology here sounds a bit like a contract is either private or divulged but that doesn’t seem to be the case afaik. `oldPrivateAssets` are just all assets returned by the ACS query and the divulged ones are a subset of that. If that is the case and I’m not missing something, I would suggest to just drop the `private` prefix here and in the Haskell code.

Reply:
Makes sense, thanks. Addressed by https://github.com/digital-asset/daml/pull/6376/commits/fa910bde5a6d264a8e780055155e55e63136d442.